README.md

```diff
@@ -71,7 +71,7 @@ if you do not have the GitHub CLI installed, you can use the following command:
 
 ```bash
 # assuming project name is: my-new-project
-npx degit noclocks/template-react-ts-starter my-new-project
+pnpx degit noclocks/template-react-ts-starter my-new-project
```

Comment:
@phoward38 pretty sure `pnpx` is deprecated and the new method is via the [`pnpm dlx`](https://pnpm.io/cli/dlx) command:

```bash
pnpm dlx degit noclocks noclocks/template-react-ts-starter my-new-project
```

Reply:
You're right